src/com/raghav/javabasics/day4/CountOccurrencesOfCharInString.java

```diff
@@ -0,0 +1,32 @@
+package com.raghav.javabasics.day4;
+//Write a java program to count occurrences of each character in String in java.
+
+public class CountOccurrencesOfCharInString {
+    public static void main(String[] args) {
+        CountOccurrencesOfCharInString result = new CountOccurrencesOfCharInString();
+        result.charCount("abc");
```

Comment:
Output: 
a: 1  b: 1  c: 1  
Array is Empty/Null
Array is Empty/Null
0: 4  c: 5  
#: 1  $: 3  %: 2  &: 2  (: 1  ^: 1  a: 2  c: 1  s: 1  
 : 8  a: 2  b: 2  c: 1  d: 2  h: 1